src/config/store/devtools.ts

```diff
@@ -0,0 +1,9 @@
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { environment } from '../../environments/environment';
+
+export const StoreDevModules = [
```

Comment:
Could we add a comment here to say: "This enables Redux DevTools by default.  Note: this is overridden in production by devtools.prod.ts".

Again, I'm not sure these settings will be easy to understand, so at least some basic comments would be worthwhile.